charts/fybrik/values.yaml

```diff
@@ -160,16 +180,16 @@ manager:
 
   podAnnotations: {}
 
+  # Pod Security Context. If set, the fields of podSecurityContext override
+  # the equivalent fields of .Values.global.podSecurityContext.
+  # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context
   podSecurityContext: {}
-    # fsGroup: 2000
 
-  securityContext: {}
-    # capabilities:
-    #   drop:
-    #   - ALL
-    # readOnlyRootFilesystem: true
-    # runAsNonRoot: true
-    # runAsUser: 1000
+  # Container Security Context. If set, the fields of containerSecurityContext override
+  # the equivalent fields of .Values.global.containerSecurityContext.
+  # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1
+  containerSecurityContext:
+    readOnlyRootFilesystem: false
 
   resources: {}
```

Comment:
Typically default request and limits are defined for the workload. Setting default resource limits help to minimize the impact of memory leaks and other ways that pods and containers can impact on other components.